Bumping dependencies

Sauloxd opened the pull request:
Hi All! :)
I've been talking to @joeldenning and as the start for my open source contributions, I'm helping to do some chores on this repo!
I don't know the best way to bump dependencies, but I've used `yarn upgrade-interactive --latest`, I wonder if there's a better way?

### My testing routine:
- `yarn start` and `yarn build` after webpack/@babel/copyfiles/rimraf
- Runtime in webpack-dev-server and the dist folder (using `http-server ./dist`) after react, kremling
- `yarn lint` and `yarn prettier` after eslint
- Do a single commit after husky

### Important notes
Bumping `husky` to @4 comes with a node requirement of >= 10 (and Git >= 2.13.0)
Maybe it's better to add node@>=10 as the engine on package.json?

frehner commented:
Looks good to me. 

I think adding the engines field to package.json is a good idea too. Do you mind adding that?

joeldenning commented:
Thanks for your work on this - yes, upgrading to the latest packages is the right thing to do when we can. I usually use `yarn upgrade-interactive --latest`.

Could you also upgrade the dependencies in https://github.com/single-spa/single-spa-playground/blob/master/index.html and https://github.com/single-spa/single-spa-playground/blob/master/src/html-file/html-file.component.js to their latest versions?

Sauloxd commented:
### Updates:
- added node >=10 in package.json
- Updated single-spa@5.1.1 in index.html for single-spa-playground
- Updated single-spa@5.1.1 and systemjs@6.2.5
